sql/firefox_accounts/fxa_amplitude_email_clicks/view.sql

```diff
@@ -0,0 +1,190 @@
+/*
+
+See https://bugzilla.mozilla.org/show_bug.cgi?id=1633918
+
+*/
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.firefox_accounts.fxa_amplitude_email_clicks`
+AS
+WITH hmac_key AS (
+  SELECT
+    AEAD.DECRYPT_BYTES(
+      (SELECT keyset FROM `moz-fx-dataops-secrets.airflow_query_keys.fxa_prod`),
```

Comment:
I'm realizing now we most likely won't be able to publish this as a view due to permissions. Airflow is able to query this table in dataops-secrets and the tables in the CDP project, but we have no guarantee that Jenkins will have those same permissions. I believe view publishing will fail without the user creating the view having access to the target tables.

Reply:
I think I'll exempt this from publishing, and then I'll have the Airflow DAG handle running this query file each time before querying it.

Reply:
But that won't work because the `bigquery-etl` stuff executes in kube with different permissions than the operators we use in the Amplitude export DAGs...